examples/hosts/iframe-host/src/inframehost.ts

```diff
@@ -37,6 +37,8 @@ class ProxyRuntime implements IRuntime {
     }
     async setConnectionState(connected: boolean, clientId?: string) {
     }
+    async raceToSummarize() {
+    }
     async stop(): Promise<IRuntimeState> {
```

Comment:
could this just become an option on stop, rather than a new method?

Reply:
i see a deprecated not on stop. maybe it could be part of the existing snapshot method?